### DISCUSSION TITLE
[mini-css-extract-plugin] is it possible to merge non-entry css chunk to entry chunk ?

### DISCUSSION THREAD

**Discussion by taozhou-glean** [webpack/webpack#16938] [Q&A] [answered]
A bit more context here, we are using `webpackChunkName` to split some lazy loaded react components into their own chunks, it works great, most of the components will have their own js / css while keeping our main entry (`bundle`) lean. And due to some client side optimization we are doing, we are exploring a way to merge some of the css into `bundle` even before loading that chunk.

Wondering if there is a way to support that ? thanks!

**Comment by alexander-akait** [comment]
You can partially achive it using splutChunks or you can create own plugin

**Reply by taozhou-glean** [reply]
thanks! can you be a bit more specific ? only touched `splitChunks` once before... tried a little bit locally and more questions than answers ;)  any similar examples done with `splitChunks` that I can take a look ? the https://webpack.js.org/plugins/split-chunks-plugin/ contains very limited basic examples..

**Reply by alexander-akait** [reply]
Maybe you can create small reproducibl repo and provide what you have and what your expected?

**Reply by taozhou-glean** [reply, marked as answer]
sorry, i should update here, I do found a way to make it work with splitChunks ;)

```
 bundleStyles: {
          chunks: (chunk) => {
            return ['bundle', 'another-module'].includes(chunk.name)
          },
          enforce: true,
          name: 'bundle',
          type: 'css/mini-extract',
},
``
